Linux_cmake/nheqminer_cpu/CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+project(nheqminer_cpu)
+cmake_minimum_required(VERSION 2.8)
+
+#aux_source_directory(. SRC_LIST)
+#add_executable(${PROJECT_NAME} ${SRC_LIST})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+# LINUX
+if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
I think this let's only `g++`, but `clang` works fine too: https://github.com/warmcat/libwebsockets/issues/352